### DISCUSSION TITLE
读光模型结果格式不match&onnx模型未提供

### DISCUSSION THREAD

**Discussion by JinShchen** [RapidAI/TableStructureRec#1] [Q&A]
您好，我最近也在做表格的抽取，也是用的读光模型和paddlepaddl的ocr，但对复杂表格的支持还存在一些bad case，看到此仓库想测试一下效果。目前有一些疑问：

1.读光模型的output的polygons的size为[N, 8]，如官网所示。
但是本仓库的里的code，对polygons的取值方式为：
`polygons[:, 0, 1]`
想问格式是改掉了吗？

2.另外，转为onnx后的模型会提供吗？

**Comment by SWHL** [comment]
> 1.读光模型的output的polygons的size为[N, 8]，如官网所示。
但是本仓库的里的code，对polygons的取值方式为：
polygons[:, 0, 1]
想问格式是改掉了吗？

请具体说明你这里用的哪个读光模型，有两个一个是有线表格识别，一个是无线表格识别，我懵逼中

> 2.另外，转为onnx后的模型会提供吗？

ONNX模型已经提供了，pypi上的whl包里就有对应的ONNX模型的

**Reply by JinShchen** [reply]
是有线表格

**Reply by Joker1212** [reply]
有线表格后面修了一点bug，然后弄成和无线表格一样的格式了，也是为了统一后续流程，然后目前默认是使用的v2的有线模型，比读光有线模型精度更高

**Comment by SWHL** [comment]
问题1，我不太记得了。有些久远了。

如果你发现的确不一样，那可能会是改了。

**Comment by ashura777** [comment]
raise TypeError("Unable to load from type '{0}'".format(type(path_or_bytes)))
TypeError: Unable to load from type '<class 'pathlib.WindowsPath'>'
出现这个错误怎么办

**Reply by SWHL** [reply]
请提供一个复现代码和运行环境

**Reply by ashura777** [reply]
我的是python3.7的是这个原因吗？是新人不是很多，

**Reply by Joker1212** [reply]
得用python3.8以上的，最好是3.10，3.7太低了

**Reply by ashura777** [reply]
是吗？但是
engine = RapidOCR()

img_path = 'foto/1.jpg'
result, elapse = engine(img_path)
是可以运行的，我的是表格图片，所以想使用表格识别

**Reply by Joker1212** [reply]
emmm,ocr的库兼容性好很多，但是这边确实3.7有点基础包都支持不了了
